packages/client/src/runtime/core/engines/data-proxy/utils/request.ts

```diff
@@ -142,3 +138,45 @@ async function nodeFetch(url: string, options: RequestOptions = {}): Promise<Req
 
 // trick to obfuscate require from bundlers, useful for Vercel Edge
 const include = typeof require !== 'undefined' ? require : () => {}
+
+class NodeHeaders {
```

Comment:
Polyfill written by the features team to make our node-fetch actually compliant with the real fetch when it comes to header management.